webpack/stories/components/Code/Code.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
hmm.. should these be part of the pr?

Reply:
Why not?
Notice it's living under `webpack/stories/component/` and it helps to show code in the storybook.

Reply:
I needed it for the storybook. It's first time where we're using it so I think it's valid to add that. I can make it a separate commit if you want.